keys-core/src/main/java/io/kidsfirst/keys/core/model/Secret.java

```diff
@@ -28,17 +28,17 @@
   @DynamoDBHashKey(attributeName = "userID")
```

Comment:
This is one is on me, but I think I broke with convention when I named it `userID` instead of `userId`. Mind fixing?